pkg/bindings/test/pods_test.go

```diff
@@ -57,6 +57,11 @@ var _ = Describe("Podman pods", func() {
 		podSummary, err := pods.List(bt.conn, nil)
 		Expect(err).To(BeNil())
 		Expect(len(podSummary)).To(Equal(1))
+
+		// Start the pod
+		_, err = pods.Start(bt.conn, newpod)
```

Comment:
Eek. This looks like a but in start for containers - I don't think this should be required

Reply:
A bug, even

Reply:
I'm fine merging for now, but we should definitely fix this.